packages/backend/index.js

```diff
@@ -126,10 +125,60 @@ app.post("/challenges", async function (request, response) {
   }
 });
 
+async function setChallengeStatus(userAddress, challengeId, newStatus, comment) {
+  const userRef = await database.collection("users").doc(userAddress);
+  const user = await userRef.get();
+  const existingChallenges = user.get("challenges");
+  existingChallenges[challengeId] = {
+    ...existingChallenges[challengeId],
+    status: newStatus,
+    reviewComment: comment != null ? comment : "",
+  };
+  await userRef.set({ challenges: existingChallenges });
+}
+
+app.patch("/challenges", async (request, response) => {
+  // ToDo. Auth. Only admins
+  const { userAddress, challengeId, newStatus, comment, test } = request.body.params;
```

Comment:
I think we can remove that `test` right?

Just tested all your changes and everything works great... good job!! ;)

Reply:
Fixed it on https://github.com/moonshotcollective/scaffold-directory/pull/32/commits/7b7cddc78447c2cb6d50db3c63db11237e0effe2